src/components/shared/timeline-card.json

```diff
@@ -0,0 +1,28 @@
+{
+  "collectionName": "components_shared_timeline_cards",
+  "info": {
+    "displayName": "TimelineCard"
+  },
+  "options": {},
+  "attributes": {
+    "Title": {
```

Comment:
Suggestion:
Let's change case of properties names. `Title` => `title`. The same for other properties.

Reply:
Renamed